src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldHaveCorrectSuffixTests.cs

```diff
@@ -1452,6 +1452,283 @@ End Sub
 End Class");
         }
 
+        [Theory, WorkItem(1818, "https://github.com/dotnet/roslyn-analyzers/issues/1818")]
+        // Following values are ok
+        [InlineData("dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader->{}")]
+        [InlineData("dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader-> {}")]
+        [InlineData("dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader->{} ")]
+        [InlineData("dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader-> {} ")]
+        // Following values are not ok
+        [InlineData("dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader->{ }")]
+        [InlineData("dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader-> { }")]
+        [InlineData("dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader->{ } ")]
+        [InlineData("dotnet_code_quality.CA1710.additional_required_suffixes = T:System.Data.IDataReader-> { } ")]
```

Comment:
Not sure if you were expecting these cases to be supported or not.

Reply:
All of these values should work. We should trim the leading and trailing whitespaces in each value.

Reply:
I'm already doing a trim (start and end). The values I have highlighted here are the ones for which the user has put space between the curly braces. Is it something we want to handle? If so how many spaces shall be allowed?

Reply:
IMO All spaces should be ignored. I think you can handle trimmed values starting and ending with curliest and any number of spaces in between as empty value.